web/src/muted_users_ui.js

```diff
@@ -59,7 +58,6 @@ export function rerender_for_muted_user() {
     // If a user is (un)muted, we want to update their avatars on the Recent Conversations
     // participants column.
     recent_view_ui.complete_rerender();
-    inbox_ui.update();
 }
```

Comment:
This one might be necessary; the comment seems like it will apply when we add a "senders" column. 

Muting does have the side effect of marking every message the user sent as read, though, so maybe that part of the operation will live-update the user away for us... I think if we don't have the call here, we should have a comment.

Reply:
Added.

Reply:
change to this before merging:

``` diff
+    // In theory, we might need to do inbox_ui.update here. But
+    // because muting a user marks every message the user has sent as
+    // read, it will update the inbox UI, if necessary through that
+    // mechanism.
```

I think though that it's possible we do need a call here -- if we have special rendering for a group DM conversation containing a muted user (But where someone else was the sender). What do we do in the recent view with those?